Source/MonoGame.Extended/Graphics/PrimitiveTypeExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace MonoGame.Extended.Graphics
+{
+    public static class PrimitiveTypeExtensions
+    {
+        public static int GetPrimitiveCount(this PrimitiveType primitiveType, int verticesOrIndicesCount)
+        {
+            switch (primitiveType)
+            {
+                case PrimitiveType.LineStrip:
+                {
```

Comment:
Yeah, fair enough. I can't expect everyone to have exactly the same coding style. Overall I like the way you do things. I just wanted to mention it so you know I'm actually doing a code review ;)